com.unity.ml-agents/Runtime/Agent.cs

```diff
@@ -54,15 +53,15 @@ internal struct AgentAction
         public float[] vectorActions;
     }
 
-
     /// <summary>
-    /// Agent Monobehavior class that is attached to a Unity GameObject, making it
+    /// Agent MonoBehavior class that is attached to a Unity GameObject, making it
```

Comment:
Unity's MonoBehaviour class is spelling the European way with a 'u'
```suggestion
    /// Agent MonoBehaviour class that is attached to a Unity GameObject, making it
```